src/mpris.ts

```diff
@@ -238,7 +238,7 @@ export class MPRIS extends DBusInterface {
   private app = Gtk.Application.get_default()!;
 
   constructor(public stream: APMediaStream) {
-    super("org.mpris.MediaPlayer2.Decibels", "/org/mpris/MediaPlayer2");
+    super("org.mpris.MediaPlayer2." + pkg.name, "/org/mpris/MediaPlayer2");
```

Comment:
JS template literals would be better in this case

Reply:
Could you point me to a place where these are used?

Reply:
Sorry I'm not on PC right now, but they look like

```js
const are = `Hello, ${username}`
```

Reply:
Done, is this what you meant?

Reply:
yep that's what I meant. I'm gonna test this asap